src/features/unified-share-modal/messages.js

```diff
@@ -120,6 +120,11 @@ const messages = defineMessages({
         description: 'Label for "Message" text box to email the Shared Link',
         id: 'boxui.unifiedShare.message',
     },
+    suggestedCollabsTitle: {
+        defaultMessage: 'Suggested:',
```

Comment:
Should `:` be in the message? VS having it in the render()

Reply:
I previously asked the Moji team about this. According to them, it is best to include colon in the translation.